public/app/types/explore.ts

```diff
@@ -38,14 +38,14 @@ export interface ExploreState {
    * True if time interval for panels are synced. Only possible with split mode.
    */
   syncedTimes: boolean;
-  /**
-   * Explore state of the left split (left is default in non-split view).
-   */
-  left: ExploreItemState;
-  /**
-   * Explore state of the right area in split view.
-   */
-  right?: ExploreItemState;
+
+  // This being optional wouldn't be needed with noUncheckedIndexedAccess set to true, but it cause more than 5k errors currently.
+  // In order to be safe, we declare each item as pssobly undefined to force existence checks.
+  // This will have the side effect of also forcing undefined checks when iterating over this object entries, but
+  // it's better to error on the safer side.
+  panes: {
+    [paneId in ExploreId]?: ExploreItemState;
```

Comment:
Can I convince you to make `ExploreId` just a string, and then use string constants (rather than enums) higher up in the stack if you need it to be a specific value?

Reply:
yeah, this will ultimately happen in an upcoming PR, we wanted to do this in this PR but ended up leaving the ExploreId enum as it'd make it easier to catch errors in the upcoming refactoring